internal/hcl/block.go

```diff
@@ -1046,7 +1061,7 @@ func (b *Block) values() cty.Value {
 	if b.Type() == "provider" {
 		for _, child := range b.Children() {
 			key := child.Type()
-			if key == "dynamic" || key == "depends_on" {
```

Comment:
I removed this from here since it's not needed and `depends_on` is an attribute not a child block.